#! Param 1
mass_arm: [5,0,0,0,0,0,
           0,5,0,0,0,0,
           0,0,5,0,0,0,
           0,0,0,5,0,0,
           0,0,0,0,5,0,
           0,0,0,0,0,5]
damping_arm: [5,0,0,0,0,0,
              0,5,0,0,0,0,
              0,0,5,0,0,0,
              0,0,0,5,0,0,
              0,0,0,0,5,0,
              0,0,0,0,0,5]
stiffness_coupling: [10,0,0,0,0,0,
                     0,10,0,0,0,0,
                     0,0,10,0,0,0,
                     0,0,0,10,0,0,
                     0,0,0,0,10,0,
                     0,0,0,0,0,10]

# #! Param 2
# mass_arm: [10,0,0,0,0,0,
#            0,10,0,0,0,0,
#            0,0,10,0,0,0,
#            0,0,0,10,0,0,
#            0,0,0,0,10,0,
#            0,0,0,0,0,10]
# damping_arm: [2,0,0,0,0,0,
#               0,2,0,0,0,0,
#               0,0,2,0,0,0,
#               0,0,0,2,0,0,
#               0,0,0,0,2,0,
#               0,0,0,0,0,2]
# stiffness_coupling: [5,0,0,0,0,0,
#                      0,5,0,0,0,0,
#                      0,0,5,0,0,0,
#                      0,0,0,5,0,0,
#                      0,0,0,0,5,0,
#                      0,0,0,0,0,5]

base_link: "base"
end_link: "tool0"
# [position (x, y, z)  orientation (x, y, z, w)]
desired_pose:  [0.14,0.660,0.520,       1, 0, 0.0, 0.0]
test_desired_pose:  [0.14,0.660,0.520,        1, 0, 0.0, 0.0]
# desired_pose:  [-0.4,0.0,0.80,       0.5, 0,0.5,-0.5]

# desired_pose:  [0.5,0.0,0.50,       0.477, -0.480, 0.646, -0.353]
# workspace_limits: [-2, 2, -2, -2, 0.10, 2]
workspace_limits: [-1.50, 1.50, -2, 2, 0.20, 2]
wrench_external_desired: [0,0,0,0,0,0]

arm_max_vel: 1.5
arm_max_acc: 1.0
platform_max_vel: 0.1
platform_max_acc: 0.2